### PR TITLE
Instrumentation: adding tags to data tracing

### PR DESCRIPTION
**Pull request by jeremyf**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [X] Optimization

## Description

Prior to this commit, the feed's class was part of the resource.  That
resulted in a more opaque view of things.

With this change, we're adding tags to the view so we should have a
better sense of filtering.

## Related Tickets & Documents

Related to #15240

## QA Instructions, Screenshots, Recordings

None

### UI accessibility concerns?

None

## Added/updated tests?

- [X] No, and this is why: changing data dog reporting

## [Forem core team only] How will this change be communicated?

- [X] I'm not sure how best to communicate this change and need help

### CODE REVIEW

**Review by jgaskins — APPROVED** [review]
Nice!

File: app/controllers/stories/feeds_controller.rb [hunk 1]
```diff
@@ -43,8 +43,10 @@ def signed_in_base_feed
              else
                Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
              end
-      Datadog.tracer.trace("feed.query", span_type: "db",
-                                         resource: "#{self.class}.#{__method__}.#{feed.class.to_s.dasherize}") do
+      Datadog.tracer.trace("feed.query",
+                           span_type: "db",
+                           resource: "#{self.class}.#{__method__}",
+                           tags: { feed_class: feed.class.to_s.dasherize }) do
```
Comment (juliannatetreault) [inline app/controllers/stories/feeds_controller.rb:49]: TIL `dasherize`!

**Review by juliannatetreault — APPROVED** [review]
It looks like the license/CLA check might be stuck, but that aside, this LGTM 👀

File: app/controllers/stories/feeds_controller.rb (hunk 1 above)
Comment (msarit) [reply app/controllers/stories/feeds_controller.rb:49]: same!

**Review by msarit — APPROVED** [review]

**Pull request merged**